provider/cmd/pulumi-resource-cloudflare/schema.json

```diff
@@ -2951,14 +3014,14 @@
                 },
                 "sessionAffinity": {
                     "type": "string",
-                    "description": "Specifies the type of session affinity the load balancer should use unless specified as `none` or `\"\"` (default). With value `cookie`, on the first request to a proxied load balancer, a cookie is generated, encoding information of which origin the request will be forwarded to. Subsequent requests, by the same client to the same load balancer, will be sent to the origin server the cookie encodes, for the duration of the cookie and as long as the origin server remains healthy. If the cookie has expired or the origin server is unhealthy then a new origin server is calculated and used. Value `ip_cookie` behaves the same as `cookie` except the initial origin selection is stable and based on the client's IP address. Available values: `\"\"`, `none`, `cookie`, `ip_cookie`, `header`. Defaults to `none`.\n"
+                    "description": "Configure attributes for session affinity.\n"
```

Comment:
This looks like it might be a loss of information?

Reply:
That is actually the correct doc string for that field. You can see the upstream doc here:
https://github.com/cloudflare/terraform-provider-cloudflare/blob/8008476152ba7f6e8d69dfc085a4dc2264c8f5b8/docs/resources/load_balancer.md#L199

The previous (longer) doc string is for a field of the same name at the top level:
https://github.com/cloudflare/terraform-provider-cloudflare/blob/8008476152ba7f6e8d69dfc085a4dc2264c8f5b8/docs/resources/load_balancer.md#L94-L95